airbyte-integrations/connectors/source-shopify/source_shopify/source.py

```diff
@@ -129,7 +129,7 @@ def filter_records_newer_than_state(self, stream_state: Mapping[str, Any] = None
         # Getting records >= state
         if stream_state:
             for record in records_slice:
-                if record.get(self.cursor_field, "") >= stream_state.get(self.cursor_field):
+                if (record.get(self.cursor_field) or "") >= stream_state.get(self.cursor_field):
```

Comment:
Can `stream_state.get(self.cursor_field)` also evaluate to `None`?

Reply:
well, yes, but we configure it to not be -- we always set it in our airbyte config